samples/rabbitmq/simple/Rabbit_5/Receiver/MyEventHandler.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Receiver
+{
+    using System.Threading.Tasks;
+    using NServiceBus;
+    using NServiceBus.Logging;
+
+    public class MyEventHandler : IHandleMessages<MyEvent>
+    {
+        static ILog log = LogManager.GetLogger<MyEventHandler>();
+
+        public Task Handle(MyEvent eventMessage, IMessageHandlerContext context)
+        {
+            log.Info($"Hello from {nameof(MyEventHandler)}");
```

Comment:
Can we rename `MyEventHandler` to `TheOtherSide` ? 😂

Reply:
![](https://media.giphy.com/media/fTs2BdzhDetPwvy3BE/giphy.gif)